interface/main/finder/dynamic_finder.php

```diff
@@ -337,48 +341,41 @@ function persistCriteria(el, e) {
     $oemr_ui = new OemrUI($arrOeUiSettings);
     ?>
 </head>
-<body class="body_top">
-    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?>">
+<body>
+    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?> mt-3">
          <div class="row">
-            <div class="col-sm-12">
-                <div class="page-header">
-                    <?php echo $oemr_ui->pageHeading() . "\r\n"; ?>
-                </div>
+            <div class="col-12">
+                <?php echo $oemr_ui->pageHeading() . "\r\n"; ?>
             </div>
-        </div>
-        <div class="row">
-            <div class="col-sm-12">
+            <div class="col-12">
                 <?php if (AclMain::aclCheckCore('patients', 'demo', '', array('write','addonly'))) { ?>
-                    <button id="create_patient_btn1" class="btn btn-secondary btn-add" onclick="top.restoreSession();top.RTop.location = '<?php echo $web_root ?>/interface/new/new.php'"><?php echo xlt('Add New Patient'); ?></button>
+                    <button id="create_patient_btn1" class="btn btn-primary btn-add" onclick="top.restoreSession();top.RTop.location = '<?php echo $web_root ?>/interface/new/new.php'"><?php echo xlt('Add New Patient'); ?></button>
                 <?php } ?>
             </div>
-            </div>
-        <br />
-        <div class="row">
-            <div class="col-sm-12">
+            <div class="col-12 jumbotron mt-3 p-4">
                 <div id="dynamic"><!-- TBD: id seems unused, is this div required? -->
                     <!-- Class "display" is defined in demo_table.css -->
```

Comment:
Hi @bradymiller 

I found this comment, I try to remove the `<div id="dynamic">`,
it looks like ok. Do you recommend to remove it?

Reply:
@stu01509 It doesn't appear to go to anything yet i'll let @bradymiller be the judge of that so I don't break any JS lol

Reply:
Okay, lol